doc/sphinx-guides/source/container/running/demo.rst

```diff
@@ -140,19 +140,25 @@ One you make this change it should be visible in the copyright in the bottom lef
 Multiple Languages
 ++++++++++++++++++
 
-Generally speaking, you'll want to follow :ref:`i18n` in the Installation Guide to set up multiple languages such as English and French.
+Generally speaking, you'll want to follow :ref:`i18n` in the Installation Guide to set up multiple languages. (You need to create your own "languages.zip" file, for example.) Here will give you guidance specific to this demo tutorial. We'll be setting up a toggle between English and French.
 
-To set up the toggle between English and French, we'll use a slight variation on the command in the instructions above, adding the unblock key we created above:
+First, edit the ``compose.yml`` file and uncomment the following line:
 
-``curl "http://localhost:8080/api/admin/settings/:Languages?unblock-key=unblockme" -X PUT -d '[{"locale":"en","title":"English"},{"locale":"fr","title":"Français"}]'``
+.. code-block:: text
+
+        #-Ddataverse.lang.directory=/dv/lang
 
-Similarly, when loading the "languages.zip" file, we'll add the unblock key:
+Next, upload "languages.zip" to the "loadpropertyfiles" API endpoint as shown below. This should place files ending in ".properties" into the ``/dv/lang`` directory configured above.
```

Comment:
I'd rather this state "This will place files..." instead of "This should place files..."

Reply:
Good call. Fixed in d9214a3239.